### PR TITLE
Reduce sd_bus verbosity with bus functions

### PR DESCRIPTION
**Pull request by gtwang01**
Summary:
D-Bus functions are less verbose than sd_bus functions.
This modifies some sd_bus calls to equivalent bus calls.

Test Plan:
meson build -Dmode=developer
meson compile -C build
meson test -C build

### CODE REVIEW

**Review by bluca — APPROVED** [review]

**Comment by evverx** [comment]
@gtwang01 I wonder why the "From" field in the commit message is `{{ <{{>`?

**Comment by evverx** [comment]
@mrc0mmand on a somewhat related note I think https://github.com/systemd/systemd/blob/main/.github/workflows/coverity.yml#L31 should probably avoid using `--pretty=\"%aE\"` coming from random commit messages.

**Comment by gtwang01** [comment]
> @gtwang01 I wonder why the "From" field in the commit message is `{{ <{{>`?

I... have no idea. Does this happen often?

**Comment by evverx** [comment]
I don't think it happens very often. It should be possible to change that by running something like
```sh
git config --global user.name "firstname lastname"
git config --global user.email "realaddress@example.com"
git commit --reset-author --amend
```
and force-pushing the branch.

**Comment by evverx** [comment]
@gtwang01 thanks! FWIW looking at https://github.com/systemd/systemd/pull/24251 it appears templates of some kind were used to set up `git` and `{{` and `}}` were copied verbatim even though they should have probably been removed somewhere.

File: src/sulogin-shell/sulogin-shell.c [hunk 1]
```diff
@@ -52,14 +51,13 @@ static int start_default_target(sd_bus *bus) {
...
-                               &error,
-                               NULL,
-                               "ss", SPECIAL_DEFAULT_TARGET, "isolate");
+        r = bus_call_method(
+                bus,
+                bus_systemd_mgr,
+                "StartUnit",
+                &error,
+                NULL,
+                "ss", SPECIAL_DEFAULT_TARGET, "isolate");
```
Comment (yuwata) [inline src/sulogin-shell/sulogin-shell.c]: Please use 16ch indentation.

**Review by yuwata — CHANGES_REQUESTED** [review]
Could you drop the unnecessary fields of the commit message, e.g. test plan, reviewers, and so on.

**Comment by vcaputo** [comment]
It's surprising how much it pleases me to see PRs migrating code to the less verbose BusLocator-oriented API.

**Review by yuwata — APPROVED** [review]

**Comment by DaanDeMeyer** [comment]
> It's surprising how much it pleases me to see PRs migrating code to the less verbose BusLocator-oriented API.

These make for amazing starter tasks to get new people familiar with contributing. I'm going to be rather sad once we run out of them.

**Pull request merged**